rotkehlchen/tests/unit/test_ethereum_airdrops.py

```diff
@@ -135,6 +135,22 @@
             'decimals': 18,
         },
     },
+    'eigen': {
+        'asset_identifier': 'EIGEN_TOKEN_PRE_RELEASE',
+        'url': 'eigenfoundation.org',
+        'api_url': 'https://claims.eigenfoundation.org/clique-eigenlayer-api/campaign/eigenlayer/credentials?walletAddress={address}',
+        'amount_path': 'data/pipelines/tokenQualified',
+        'name': 'EIGEN',
+        'icon': 'eigen.svg',
+        'icon_path': 'airdrops/icons/eigen.svg',
+        'new_asset_data': {
+            'asset_identifier': 'EIGEN_TOKEN_PRE_RELEASE',
```

Comment:
will need to add `asset_identifier` field in `tests/airdrop_index_v1_schema.json` in the date repo, because right now it won't allow that field

Reply:
there are other things too, like csv_path and api_url

Reply:
also the api_url should have `={address}` in it, because we are .format()ing it in the code. Can use regex to check that in the schema.

Reply:
I've added those tests in the data repo